src/paddle.js

```diff
@@ -0,0 +1,19 @@
+(function(exports) {
+
+function Paddle(canvas) {
+  this.canvas = canvas;
+  this.paddleHeight = 60;
+  this.paddleWidth = 10;
+}
+
+Paddle.prototype.draw = function (x,y) {
+  ctx.beginPath();
```

Comment:
Following on my prior request, lines 10 - 14 will therefore need to be appended as follows:
this.ctx.beginpath(); 

I hope this is clear! Let me know if you have any questions- happy to assist.

Many thanks,

Alistair :)